### PR TITLE
Clone of the Newsweek page using the framework bootstrap

### PR DESCRIPTION
**Pull request by javierbotero**
The entire page was build using the different breakpoints of bootstrap and the classes for building the layout. Any feedback would be greatly appreciated.

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi team!
### Nice job! WOW, project looks! :+1: 

### But your project needs a few changes before it can be approved.

##### [OPTIONAL] Navigation bar
* Delete space above header
space


##### Footer

*  It contains social media icons ([link to page](https://www.newsweek.com/))
In original version you can see social media icons

icons

In your version social icons are absent
icons2

* **Footer should changes with RWD breakpoints** ([link to page](https://www.newsweek.com/))

#### Implementation details


* Kindly recommended to use Bootstrap Css classes **everywhere** for styling (for example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options)) )

screen

screen
### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

*Happy coding 😄*

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1185 @@
...
+                        </div>
+                     </div>                  
+                 </div>
+             </section>
+             <section class="row m-0 magazine">
+                 <div class="col-12 row p-0 m-0 magazine-top my-3">
+                    <div class="col-12 py-2 my-2 col-md-9 offset-md-3 d-flex justify-content-center align-items-center pt-md-2 pb-md-2 mb-md-3 border-custom">
+                         <span class="icon">N</span>
+                         <h3 class="m-0">In the magazine</h3>
+                    </div>
```
Comment (Anna-Myzukina) [inline index.html]: Line number 434: content "In the magazine" should be uppercase, kindly recommended to fix it

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
The project needs changes

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+/* change some properties from bootstrap classes */
+
+/* nav bar styles */
+
+nav {
+  background-color: #f72210;
+}
+
+nav header {
+  text-align: center;
```
Comment (euqueme) [inline css/style.css]: Use [bootstrap classes for text-align](https://getbootstrap.com/docs/4.4/utilities/text/#text-alignment) directly in your HTML file

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+}
+
+.subscribe {
+  text-align: right;
```
Comment (euqueme) [inline css/style.css]: Use [bootstrap classes for text-align](https://getbootstrap.com/docs/4.4/utilities/text/#text-alignment) directly in your HTML file

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+}
+
+.arrow-right {
+  width: 10px;
+  fill: #f72210;
+}
+
+.sign-in {
+  font: bold 18px/14px "Roboto Condensed", sans-serif;
+  text-transform: uppercase;
```
Comment (euqueme) [inline css/style.css]: Use [bootstrap classes for text-transform](https://getbootstrap.com/docs/4.4/utilities/text/#text-transform) directly in your HTML file

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,332 @@
...
+}
+
+.subscribe-button {
+  background-color: white;
+  font: bold 18px/14px 'Roboto Condensed', sans-serif;
+  color: #f72210;
+  text-transform: uppercase;
```
Comment (euqueme) [inline css/style.css]: Use [bootstrap classes for text-transform](https://getbootstrap.com/docs/4.4/utilities/text/#text-transform) directly in your HTML file

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,332 @@
...
+#edit-search-block-form--4 {
+  position: absolute;
+  top: 14%;
+}
+
+/* main styles */
+
+.tag-news {
+  font: bold 16px/16px "Roboto Condensed", sans-serif;
+  text-transform: uppercase;
```
Comment (euqueme) [inline css/style.css]: Use [bootstrap classes for text-transform](https://getbootstrap.com/docs/4.4/utilities/text/#text-transform) directly in your HTML file

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,332 @@
...
+  padding-bottom: 11px;
+  color: rgba(3, 3, 3, 0.4);
+}
+
+.by {
+  font-size: 14px;
+  line-height: 21px;
+  color: #555;
+  text-transform: uppercase;
```
Comment (euqueme) [inline css/style.css:113]: Use [bootstrap classes for text-transform](https://getbootstrap.com/docs/4.4/utilities/text/#text-transform) directly in your HTML file

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,332 @@
...
+
+.wrapper {
+  /* overflow: hidden; */
+  max-width: 1310px;
+  background-color: white;
+}
+
+.icon {
+  background-color: #f72210;
+  text-align: center;
```
Comment (euqueme) [inline css/style.css:134]: Use [bootstrap classes for text-align](https://getbootstrap.com/docs/4.4/utilities/text/#text-alignment) directly in your HTML file

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,332 @@
...
+  color: white;
+  font: bold 16px/16px "Roboto Condensed", sans-serif;
+  width: 29px;
+  height: 28px;
+  display: block;
```
Comment (euqueme) [inline css/style.css:139]: Use [bootstrap classes for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file

File: css/style.css [hunk 10]
```diff
@@ -0,0 +1,332 @@
...
+}
+
+.btn-travel {
+  display: inline-block;
```
Comment (euqueme) [inline css/style.css:143]: Use [bootstrap classes for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file

File: css/style.css [hunk 11]
```diff
@@ -0,0 +1,332 @@
...
+  position: absolute;
+  left: 94%;
+  top: 86px;
+  z-index: 9;
+}
+
+.menu li {
+  border-left: 1px solid rgb(217, 217, 217);
+  flex-grow: 1;
+  display: flex;
```
Comment (euqueme) [inline css/style.css:68]: Use [bootstrap classes for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file

File: css/style.css [hunk 12]
```diff
@@ -0,0 +1,332 @@
...
+  #Layer_2 {
+    width: 396px;
+    height: 60px;
+    margin: 10px auto 0;
+  }
+}
+
+@media screen and (max-width: 992px) {
+  .logo-shape {
+    display: block;
```
Comment (euqueme) [inline css/style.css:90]: Use [bootstrap classes for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file

File: css/style.css [hunk 13]
```diff
@@ -0,0 +1,332 @@
...
+}
+
+/* End same sules for various elements */
+
+.categories-mini {
+  height: 33.3%;
+}
+
+#Layer_1 {
+  display: block;
```
Comment (euqueme) [inline css/style.css:329]: Use [bootstrap classes for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file

File: css/style.css [hunk 14]
```diff
@@ -0,0 +1,332 @@
...
+    font-size: 14px;
+  }
+}
+
+.date {
+  font: bold 18px/14px "Roboto Condensed", sans-serif;
+}
+
+.righ-data {
+  position: relative;
```
Comment (euqueme) [inline css/style.css:41]: Use [bootstrap classes for position](https://getbootstrap.com/docs/4.4/utilities/position/) directly in your HTML file

File: css/style.css [hunk 15]
```diff
@@ -0,0 +1,332 @@
...
+}
+
+.red-shape {
+  width: 18%;
+  height: 16px;
+  background-color: red;
+  transform: skew(50deg);
+  position: absolute;
```
Comment (euqueme) [inline css/style.css:49]: Use [bootstrap classes for position](https://getbootstrap.com/docs/4.4/utilities/position/) directly in your HTML file

File: css/style.css [hunk 16]
```diff
@@ -0,0 +1,332 @@
...
+  left: 78%;
+  top: 86px;
+  z-index: 10;
+}
+
+.red-shape-square {
+  width: 6%;
+  height: 16px;
+  background-color: red;
+  position: absolute;
```
Comment (euqueme) [inline css/style.css:59]: Use [bootstrap classes for position](https://getbootstrap.com/docs/4.4/utilities/position/) directly in your HTML file

File: css/style.css [hunk 17]
```diff
@@ -0,0 +1,332 @@
...
+@media screen and (max-width: 992px) {
+  .logo-shape {
+    display: block;
+    width: 132px;
+    height: 20px;
+  }
+}
+
+#edit-search-block-form--4 {
+  position: absolute;
```
Comment (euqueme) [inline css/style.css:97]: Use [bootstrap classes for position](https://getbootstrap.com/docs/4.4/utilities/position/) directly in your HTML file

File: css/style.css [hunk 18]
```diff
@@ -0,0 +1,332 @@
...
+  margin-bottom: 6px;
+}
+
+.tag {
+  color: #f72210;
+  font: bold 17px/16px "Roboto Condensed", sans-serif;
+  text-transform: uppercase;
+  padding: 8px 15px 9px;
+  background-color: rgba(255, 255, 255, 0.9);
+  position: relative;
```
Comment (euqueme) [inline css/style.css:157]: Use [bootstrap classes for position](https://getbootstrap.com/docs/4.4/utilities/position/) directly in your HTML file

File: css/style.css [hunk 19]
```diff
@@ -0,0 +1,332 @@
...
+  bottom: 30px;
+}
+
+.sponsored {
+  color: #3581f4;
+  font: bold 17px/16px "Roboto Condensed", sans-serif;
+  text-transform: uppercase;
+  padding: 8px 15px 9px;
+  background-color: rgba(255, 255, 255, 0.9);
+  position: relative;
```
Comment (euqueme) [inline css/style.css:167]: Use [bootstrap classes for position](https://getbootstrap.com/docs/4.4/utilities/position/) directly in your HTML file

File: css/style.css [hunk 20]
```diff
@@ -0,0 +1,332 @@
...
+
+@media screen and (max-width: 991px) {
+  .sponsored-right {
+    margin: 0 30%;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .more-stories {
+    position: sticky;
```
Comment (euqueme) [inline css/style.css:180]: Use [bootstrap classes for position](https://getbootstrap.com/docs/4.4/utilities/position/) directly in your HTML file

File: css/style.css [hunk 21]
```diff
@@ -0,0 +1,332 @@
...
+    top: -1000px;
+  }
+
+  .sticky-left {
+    position: sticky;
```
Comment (euqueme) [inline css/style.css:185]: Use [bootstrap classes for position](https://getbootstrap.com/docs/4.4/utilities/position/) directly in your HTML file

File: css/style.css [hunk 22]
```diff
@@ -0,0 +1,332 @@
...
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.icon-slide {
+  position: relative;
```
Comment (euqueme) [inline css/style.css:257]: Use [bootstrap classes for position](https://getbootstrap.com/docs/4.4/utilities/position/) directly in your HTML file

File: css/style.css [hunk 23]
```diff
@@ -0,0 +1,332 @@
...
+    width: 30%;
+  }
+}
+
+.picture-end {
+  background: url("/assets/coronavirus.jpg") no-repeat;
+}
+
+.positionSpecial1 {
+  position: relative;
```
Comment (euqueme) [inline css/style.css:292]: Use [bootstrap classes for position](https://getbootstrap.com/docs/4.4/utilities/position/) directly in your HTML file

File: css/style.css [hunk 24]
```diff
@@ -0,0 +1,332 @@
...
+  left: 22%;
+  top: -6%;
+}
+
+.positionSpecial2 {
+  position: relative;
```
Comment (euqueme) [inline css/style.css:298]: Use [bootstrap classes for position](https://getbootstrap.com/docs/4.4/utilities/position/) directly in your HTML file

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

#### Hello @javierbotero :handshake: 

### Amazing job the amount of detail is impressive :medal_sports: there are some changes to make yes but they are all very minor:

**Hamburger menu**

- [x] Add a hamburger menu in the navbar for your small and medium screen sizes

 - Yours 

image

- Original

image

**Preview magazine**

- [x] Increase the height of the preview magazine in small screens size also the IN THE MAGAZINE title has to come after the preview magazine and not on top

- Yours

image


- Original

image

**SUBSCRIBE and NEWSLETTER**

- [x] change the SUBSCRIBE and NEWSLETTER titles to have a red font color 

- Yours

image

- Original

image

**Bootstrap usage**

- [x] Your CSS file still have selectors that can be replaced with bootstrap classes, I have left comments where selectors have to be removed.

That's it! :raised_hands:  

### Its been a true blast :guitar:  to review this project, Please :pray:  make the changes and submit back for a code review :rocket: keep up the good work! :muscle: 

Happy coding! :woman_technologist: 

PROJECT_SATUS: REQUESTED CHANGES :stop_sign:

File: css/style.css (hunk 20 above)
Comment (javierbotero) [reply css/style.css:180]: Hello @euqueme, first of all, thanks for your review, specifically for this point I decided to let the sticky in my css due to they are not supported to be responsive using bootstrap classes 

> Quick positioning classes are available, though they are not responsive.> 

you can find the documentation here:  

`https://getbootstrap.com/docs/4.4/utilities/position/`

Comment (euqueme) [reply css/style.css:180]: I agree, thank you for your comment :1st_place_medal:

**Review by euqueme — APPROVED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

# Hii @javierbotero :wave: 

### Thank you for doing all the changes you are a true MVP! :raised_hands:  I have one last but important change you still have to do:

- [ ] Fix the social media icons in the footer the images are broken

image

that is it! 

### I'm trusting you do this change :handshake: and after merge it to the master do not request for a code review again

Happy Coding! :woman_technologist: 

PROJECT_STATUS: APPROVED :green_apple:

**Pull request merged**